CalendarApp/BookingRoom/Controllers/PostEventController.cs

```diff
@@ -17,10 +20,21 @@ public PostEventController(BookingService bookingService)
 
         }
         [HttpPost]
-        public HttpResponseMessage EventPost(CalendarEvent eventPost)
+        public HttpResponseMessage PostEvent(CalendarEvent eventPost)
```

Comment:
It is just a "same fashion" naming for all controllers: delete/update/search/post/ Event